rosbag2_storage_mcap/package.xml

```diff
@@ -14,6 +14,7 @@
   <depend>rcutils</depend>
   <depend>ros_environment</depend>
   <depend>rosbag2_storage</depend>
+  <depend>ament_index_cpp</depend>
```

Comment:
Does this also need a `find_package` line and adding to the dependency list in CMakeLists.txt?

Reply:
It built for me, but I'll add it anyway...